EasyPost.Integration/Basics.cs

```diff
@@ -0,0 +1,11 @@
+using Xunit;
+
+namespace EasyPost.Integration;
+
+public class Basics
+{
+    [Fact]
+    public void Test1()
```

Comment:
question: this "integration" test doesn't actually test anything, help me understand what the point of actually running integration tests is?

Reply:
See "Testing" section in description above.